broad-wb-perf2.json

```diff
@@ -1,19 +1,79 @@
 {
   "name": "terra",
   "owner": "broad-wb-perf2",
-  "terraform_image": "us.gcr.io/broad-dsp-gcr-public/config-render:48229e65030",
   "intent": "terra infrastructure",
   "production": false,
   "stability": "volatile",
+  "environment": "broad-wb-perf2",
+  "project": "broad-wb-perf2",
+  "region": "us-central1",
+  "resource_zone_name": "us-central1-a",
+  "dns_zone_name": "dsde-perf-broad",
+  "dns_zone": "dsde-perf.broadinstitute.org",
   "global_vars": {
+    "vault": {
+      "policy": "dsde-write"
+    },
     "env": {
       "VAULT_PATH_PREFIX": "secret/dsde/firecloud/ephemeral/broad-wb-perf2"
     },
     "vault": {
       "policy": "dsde-write"
+    },
+    "terraform": {
+      "docker_image": "us.gcr.io/broad-dsp-gcr-public/config-render:bbda8365ceccbc1",
+      "log_level": "ERROR",
+      "providers": {
+        "google": {
+          "default": {
+            "project": "broad-wb-perf2",
+            "vault_sa_path": "secret/dsde/terra/envs/common/terra-env-owner-service-account.json",
+            "region": "us-central1"
+          }
+        }
+      },
+      "backend": {
+        "vault_sa_path": "secret/dsp/service-accts/dsp-techops/terraform-state-service-account.json",
+        "state_path_prefix":"workbench",
+        "state_bucket": "broad-dsp-terraform-state"
+      }
     }
   },
   "profile_vars": {
+    "gsuite-intermediate-domain": {
+      "vault": {
+        "policy": "dsde-write"
+      },
+      "terraform": {
+        "providers": {
+          "google": {
+            "intermediate-dns-zone-delegation": {
+              "project": "broad-dsde-prod",
+              "vault_sa_path": "secret/suitable/firecloud/prod/credentials",
+              "region": "us-central1"
+            },
+            "intermediate-dns-zone": {
+              "project": "broad-dsp-techops-dev",
+              "vault_sa_path": "secret/devops/terraform/dev/dsp-techops/credentials",
+              "region": "us-central1"
+            }
+          }
+        }
+      }
```

Comment:
why is this `profile_vars.terraform` section so different between the three input json files?